### ISSUE TITLE
[Bug] Item Pickup Animation

### ISSUE THREAD

**Issue by DarianLStephens** [crazysnailboy/AdvancedHUD#5]
I've only tested this in 1.7.10, but with the new version of the mod (Great job, by the way. All the previous bugs are fixed and it's so much more usable!), when you pick up an item, the little animation that's meant to have the item bounce in size slightly, instead seems to make the item come from a bit down and to the left of the slot it goes to.
 This is minorly distracting, and perfectly deal-with-able, unlike now-fixed issues such as odd shading of blocks and semi-broken crosshair.

**Comment by crazysnailboy** [comment]
Thank you for highlighting this - it'll be resolved with the next release :)

**Comment by DarianLStephens** [comment]
There's one more thing I've noticed; I can't see how to set the crosshair back to the original after I've selected a custom one (Besides maybe resetting the entire hud). Is there something I'm missing or is there not a way to do that?

**Comment by crazysnailboy** [comment]
Yeah I'm going to do a fix for that in the next release as well. Currently the only way to reset it is to open the config.dat file in [NBTExplorer](http://www.minecraftforum.net/forums/mapping-and-modding-java-edition/minecraft-tools/1262665-nbtexplorer-nbt-editor-for-windows-and-mac) and delete the style tag under crosshair.

**Comment by DarianLStephens** [comment]
...Why did it never occur to me that the file might be in NBT? That could have saved me a bit of hassle in the past! Anyway, it's nice to know these things will be fixed. I'll be sure to report any more issues I find, if I happen to find any!
